gems/sorbet-runtime/lib/types/private/methods/_methods.rb

```diff
@@ -16,7 +16,7 @@ def self.declare_sig(mod, &blk)
     install_hooks(mod)
 
     if T::Private::DeclState.current.active_declaration
-      T::Private::DeclState.current.active_declaration = nil
+      T::Private::DeclState.current.reset!
```

Comment:
What's the difference between `active_declaration = nil` and `reset!`?

Reply:
[There is none][1], so I used `reset!` to try to be consistent.

[1]: https://github.com/sorbet/sorbet/blob/master/gems/sorbet-runtime/lib/types/private/decl_state.rb#L17